.github/scripts/conda_env_sorter.py

```diff
@@ -8,7 +8,7 @@
 
 """Sort dependencies in conda environment files."""
 # Test with
-# uv run --with pytest -- pytest -v .github/scripts/conda_env_sorter.py
+# uv run --with pytest --with "ruamel.yaml" -- pytest -v .github/scripts/conda_env_sorter.py
```

Comment:
I don't like that this has uv as a dependency.

Reply:
It doesn't, it's just an ease of use option 🤷🏻‍♂️ 

You can still run it with `python .github/scripts/conda_env_sorter.py`